src/main/scala/dotty/renderers/ScalaHtmlRenderer.scala

```diff
@@ -40,10 +41,69 @@ class ScalaHtmlRenderer(ctx: DokkaContext) extends SiteRenderer(ctx) {
     override def buildContentNode(f: FlowContent, node: ContentNode, pageContext: ContentPage, sourceSetRestriciton: JSet[DisplaySourceSet]) = {
         node match {
             case n: HtmlContentNode => withHtml(f, raw(n.body).toString)
+            case n: DocumentableList => 
+                val ss = if sourceSetRestriciton == null then Set.empty.asJava else sourceSetRestriciton
+                withHtml(f, buildDocumentableList(n, pageContext, ss).toString())
+            case n: DocumentableFilter => withHtml(f, buildDocumentableFilter.toString)
             case other => super.buildContentNode(f, node, pageContext, sourceSetRestriciton)
         }
     }
 
+    private val clazz = `class`
```

Comment:
Minor nitpick: this is a bit confusing, since one most often sees `clazz` variables when dealing with JVM reflection. The `class` attribute is also available as `cls`, so let's maybe use that instead?

Reply:
I do not have strong preference, we can go with `cls`